roles/confluent.kafka_connect/tasks/main.yml

```diff
@@ -61,6 +61,29 @@
   notify:
     - restart connect distributed
 
+- name: Ensure Plugin Dirs
+  file:
+    path: "{{item}}"
+    state: directory
+    group: "{{kafka_connect_group}}"
+    owner: "{{kafka_connect_user}}"
+    mode: '764'
+  when: item != '/usr/share/java'
+  with_items: "{{ kafka_connect.properties['plugin.path'].split(',') }}"
+  
+- name: Confluent Hub
+  include_tasks: confluent_hub.yml
+  when: kafka_connect_confluent_hub_install
+  
+- name: Installing Plugins
```

Comment:
should this be moved into the confluent_hub.yml tasks file?

Reply:
I kept the Archive based install outside of the hub task. Since they don't require the Confluent Hub to install. Thought it'd be counter to the encapsulation of work.

Reply:
ah sorry i didnt see the difference between kafka_connect_confluent_hub_plugins and kafka_connect_plugins